usvm-python/usvm-python-annotations/src/main/kotlin/org/usvm/annotations/ids/SlotId.kt

```diff
@@ -0,0 +1,69 @@
+package org.usvm.annotations.ids
+
+/*
+ * tp_hash cannot be disabled, however
+ * if you don't explicitly specify that the type HAS that slot,
+ * you will not be able to disable the tp_richcompare slot
+ * 
+ * For that reason, some slots are marked as mandatory.
+ * They cannot be disabled using mask, so they do not have
+ * a mask bit number.
+ * 
+ * The usage of swapSlotBit or setSlotBit with these slots
+ * will not have any effect on the mask.
+ */
+enum class SlotId(
+    val slotName: String,
+    val mandatory: Boolean = false
+) {
+    TpGetattro("tp_getattro"),
+    TpSetattro("tp_setattro", true),
```

Comment:
The tests in the `VirtualObjectsTest.kt` showed that `tp_getattro` can be disabled, but that's not the case with `tp_setattro` . So, yeah, I'm sure. It can be made mandatory, but I tend not to add constraints unnecessarily.

Reply:
No, there was a bug in the test. Fixed that, `tp_getattro` is now mandatory too.